### PR TITLE
Fix a bunch of issues

### PR DESCRIPTION
**Pull request by 4383**
- wrong openstack generated url
- flake8 warnings
- ignore flake8 W504

Also adding the tox `venv` target to run beagle commands on virtualenv managed by tox

### CODE REVIEW

File: beagle/openstack.py [hunk 1]
```diff
@@ -22,7 +22,11 @@ def add_argument_group(self, parser):
         pass
 
     def emit_list(self, column_names, data, stdout, parsed_args):
-        fmt = 'http://git.openstack.org/cgit/openstack/{Repository}/tree/{Filename}#n{Line} : {Text}\n'
+        fmt = '{base_url}/{repo}'.format(
+            base_url='https://opendev.org',
```
Comment (4383) [inline beagle/openstack.py:26]: url was not valid

File: beagle/openstack.py [hunk 2]
```diff
@@ -32,7 +36,7 @@ def emit_list(self, column_names, data, stdout, parsed_args):
 
             if parsed_args.context_lines:
                 before = row_d['Before'].machine_readable()
-                write_lines_with_offset(
+                write_lines_with_offset(  # noqa
```
Comment (4383) [inline beagle/openstack.py:39]: linter warn here, I'm not sure if we should drop this call

File: beagle/openstack.py [hunk 3]
```diff
@@ -43,7 +47,7 @@ def emit_list(self, column_names, data, stdout, parsed_args):
             stdout.write(fmt.format(**row_d))
 
             if parsed_args.context_lines:
-                write_lines_with_offset(
+                write_lines_with_offset(  # noqa
```
Comment (4383) [inline beagle/openstack.py:50]: linter warn here, I'm not sure if we should drop this call

File: doc/source/user/index.rst [hunk 4]
```diff
@@ -67,8 +67,9 @@ To produce links to the source on the OpenStack git server, use the
 .. code-block:: console
 
    $ beagle --debug search -f link --repo openstack/oslo.config 'class ConfigOpts'
-   http://git.openstack.org/cgit/openstack/openstack/oslo.config/tree/doc/source/reference/configuration-files.rst#n10 : class ConfigOpts(object):
-   http://git.openstack.org/cgit/openstack/openstack/oslo.config/tree/oslo_config/cfg.py#n1925 : class ConfigOpts(abc.Mapping):
+   https://opendev.org/openstack/oslo.config/src/branch/master/doc/source/reference/configuration-files.rst#n10 : class ConfigOpts(object):
```
Comment (4383) [inline doc/source/user/index.rst:70]: Due to the previous url issue these examples are wrong

File: tox.ini [hunk 5]
```diff
@@ -26,7 +29,9 @@ commands =
 
 [flake8]
 # E123, E125 skipped as they are invalid PEP-8.
+# W504 skipped to give priority to W503
+# W503 = Line breaks should occur after the binary operator
 show-source = True
-ignore = E123,E125
+ignore = E123,E125,W504
```
Comment (4383) [inline tox.ini:35]: Allow us to fix a linter warning

**Comment by 4383** [comment]
These changes are now part of #5 to fix the pipeline in one place (I really prefer how gerrit manage patches).

**Comment by dhellmann** [comment]
I think I copied the travis setup into this repo from somewhere else without enabling travis, and then changed travis so it's active for all of my environments. There aren't actually any unit tests for this project, so we probably just want to run the linter job.

**Comment by dhellmann** [comment]
and of course now I see you've done that in #5

**Pull request closed without merging**